src/components/Product/Amount/Amount.tsx

```diff
@@ -1,12 +1,22 @@
 import React, { useEffect, useState } from 'react';
 import './Amount.css';
 
-export const Amount = ({ handlerAmount, defaultValue, updatedValue }) => {
+interface props {
```

Comment:
As convention interfaces are declared starting with the capital "I" next to the name of the object or variable, ej.
interface IAmount { ...